community/event-funding.tt

```diff
@@ -0,0 +1,105 @@
+[% WRAPPER layout.tt title="Event funding" handlesLayout=1 %]
+
+<div class="page-title">
+  <div>
+    <a href="[% root%]community/index.html">Community</a>
+    <span>→</span>
+  </div>
+  <h1>Event funding</h1>
+</div>
+
+<section class="community-event-funding">
+  <p>
+    The NixOS community is awesome! You guys have been generously donating
```

Comment:
The NixOS community is awesome! Our friends and members have been donating throughout the year, and now we'd like to put the money to work. We are launching a new grant program that will help our community thrive and widen. 
      
In particular, we are planning to fund meet-ups and conferences, as in-person communication can inspire developers to new ideas and experiments and lead to teaming up and building new projects.